### DISCUSSION TITLE
Why github campus expert form isn't opening? Apply button isn't working.

### DISCUSSION THREAD

**Discussion by gaurab-khanal** [community/community#46525] [GitHub Education] [answered]
Apply button for guthub campus expert isn't working

**Comment by Mr-Sunglasses** [comment]
Same the form is not working :( .

**Comment by lasr21** [comment]
Howdie, have you try here https://education.github.com/campus_experts ?

**Reply by Mr-Sunglasses** [reply]
Yes and it is not working https://education.github.com/campus_experts/status see.

**Comment by Mr-Sunglasses** [comment]
https://github.blog/changelog/2023-02-06-campus-experts-applications-move-to-github-global-campus/ @gaurab-khanal I think this blog will help.

**Reply by gaurab-khanal** [reply]
https://education.github.com/campus_experts/status 
Are you saying this link? Here is also the same problem. The apply button is not working.

**Comment by samus-aran** [comment]
Hi folks 👋 

Can you explain to us in steps what you're experiencing? This is so we can assist further.

**Reply by Mr-Sunglasses** [reply]
Hey @samus-aran , Now it is fixed. Thanks for the help !!

**Reply by samus-aran** [reply]
Fabulous @Mr-Sunglasses ! Best of luck to all applying 💃 🕺

**Comment by ShadrackOdielo** [comment, marked as answer]
hello.
it eventually started working

On Thu, Feb 9, 2023 at 5:04 AM Ashley Nicolson ***@***.***>
wrote:

> Hi folks 👋
>
> Can you explain to us in steps what you're experiencing? This is so we can
> assist further.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/46525#discussioncomment-4913231>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWFDDOXP5VNJTLDBQFSWUN3WWRGDXANCNFSM6AAAAAAUS47UFM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>

-- 
*Website: www.jkuat.ac.ke <http://www.jkuat.ac.ke>*

**Reply by samus-aran** [reply]
Yay! Wonderful news. Thanks for confirming that for us @ShadrackOdielo. Best of luck if applying 👏

**Reply by zacharias1219** [reply]
Hi guys, my link isn't opening, I'm unable to check my application status.
